packages/menu/src/MenuContainer.spec.tsx

```diff
@@ -846,6 +846,27 @@ describe('MenuContainer', () => {
       expect(fruit1).toHaveAttribute('aria-checked', 'true');
     });
 
+    it('returns normal keyboard navigation after menu closes', async () => {
+      const { getByText, getByTestId } = render(
+        <>
+          <TestMenu items={ITEMS} />
+          <button>focus me</button>
+        </>
+      );
+      const trigger = getByTestId('trigger');
+      const nextFocusedElement = getByText('focus me');
+
+      trigger.focus();
+
+      await act(async () => {
```

Comment:
With #600 I'm finding that these multi-action constructs need to be wrapped by `waitFor` rather than `act`. I'll sweep this up once it has been merged.

Reply:
Would it be more "correct" to split up each action into its own `act` so the queue flushes completely? Might be more accurate to the browser even?

Reply:
With other containers, I think we just line up the actions without the wrapped `act`. Does that generate warnings here?